automation/vars/uni07eta-adoption.yaml

```diff
@@ -0,0 +1,26 @@
+---
+vas:
+  uni07eta-adoption:
+    stages:
+      - path: examples/dt/uni07eta/control-plane/networking/nncp
```

Comment:
This file should contain basically first two stages of the original uni07eta – please add the names of the stages, so it will look exactly the same.